homeassistant/components/mqtt/__init__.py

```diff
@@ -12,10 +12,10 @@
 import os
 import socket
 import ssl
-import time
 from typing import Any, Callable, List, Optional, Union, cast  # noqa: F401
 
 import attr
+from paho.mqtt.client import MQTT_ERR_SUCCESS
```

Comment:
OK to import here?

Reply:
No, it has to be done inside a function since it's not a core requirement.

Reply:
Hmm, so one import every time the constant is used.
Maybe that's worse than just comparing to magic number 0 as before?

Reply:
Imports are cached so it's not a performance problem. It's just a bit messy.

Reply:
We could save the client module in ab instance attribute.